test/cli/fixtures/memory-leak/index.js

```diff
@@ -56,7 +56,7 @@ QUnit.module( "later thing", function() {
 
 		let snapshot = await streamToString( v8.getHeapSnapshot() );
 		let matches = snapshot.match( reHeap );
-		assert.ok( matches && matches.length, "found local Foo in heap" );
+		assert.true( matches && matches.length, "found local Foo in heap" );
```

Comment:
```
  message: "found local Foo in heap"
  severity: failed

  actual: 2
  expected: true
```

The  `matches.length` array property returns a number, which is `2` in this case.

You can keep the same loose semantics by changing it to `!!matches && matches.length > 0`, or preferably, use `strictEqual` expecting `2`.

This test interacts with the internal GC interface of Node.js, and because this changes between versions we only assert the behaviour of Node 14 and the test is skipped entirely on other versions. This is why the other builds are passing.

Reply:
Fixed. Thanks for looking into this!